.github/workflows/pypi-workflow.yml

```diff
@@ -0,0 +1,47 @@
+name: build
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Wouldn't it be interesting to only trigger a Pypi release when new git tags are added?

Reply:
that would be interesting, but i figured this workflow of "update pypi to match the source of truth: the repo" made more sense. open to being convinced either way, this is just a start

Reply:
I'm ok with this workflow, but then I do not see the point of having a separate `.github/workflows/pypi-workflow.yml ` file: wouldn't be simpler to add a "Release on Pypi" step on the existing YAML, with a conditional like `if: github.ref == 'refs/heads/master' && matrix.python-version == '3.9'`?

I'd like to avoid duplicating the CI pipeline logic :)

We could also have all the stuff related to the Python code on one pipeline (lintter, code prettifier & tests for each Python version supported) and all the other stuff on another pipeline (documentation generation & release on Pypi)

What do you think?

Reply:
i think you should send me your pypi account username so i can invite you because at that point you'll have all the access you could possibly need to do everything

Reply:
I already suggest it here: https://github.com/PyFPDF/pyfpdf/pull/41#issuecomment-757527346
;)